projects/espeak-ng/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://github.com/espeak-ng/espeak-ng"
+language: c++
+primary_contact: "sascha@brawer.ch"
+auto_ccs :
+- "p.antoine@catenacyber.fr"
+
+sanitizers:
+- address
+fuzzing_engines:
+- libfuzzer
+- afl
```

Comment:
why not honggfuzz, can add in next cl.

Reply:
CI Failed for it cf commits
- 🔴 3f1abf1
- 💚 39b3de6